api-module-library/microsoft-teams/.env.example

```diff
@@ -0,0 +1,4 @@
+TEAMS_CLIENT_ID=
+TEAMS_CLIENT_SECRET=
+REDIRECT_URI=http://localhost:3000/redirect/teams
```

Comment:
See below

Also, for MSFT I want to include `microsoft` in the name, so `microsoft-teams`